static/scss/cms/instructor.scss

```diff
@@ -0,0 +1,108 @@
+.faculty-block {
+  color: $light-black;
```

Comment:
I don't really understand "light-black". I mean, "grey-50" or something like that makes sense, but "light-black" or "dark-white" don't really make sense to me.